LICENSE.md

```diff
@@ -47,12 +54,11 @@ This license is separate from the license of any Mod it depends on and does not
 --------
 
 The User accepts the following liability:
- - The Mod is provided 'as is' with no warranties, implied or otherwise.
- - The Author is not responsible for dragon, troll, pirate, or ninja attacks.
```

Comment:
Absolutely do not remove this line. Refer to [PR#13](https://github.com/jakimfett/MMPLv2/pull/13) for the discussion of why.